doc/doxygen/references.bib

```diff
@@ -2022,3 +2022,14 @@ @article{zhang2022petscsf
 address = {Los Alamitos, CA, USA},
 month = {apr}
 }
+
+@article{zampini2016pcbddc,
```

Comment:
@stefanozampini Is this the right reference?

Reply:
yes, thanks